### DISCUSSION TITLE
I can use it only if i know other people who use it or i can find public groups inside?

### DISCUSSION THREAD

**Discussion by slrslr** [oxen-io/session-desktop#2208] [Q&A] [answered]
I am unsure if it is apparent from https://getsession.org whether in this software i have to first know contacts of other session users or there are public "groups" (kind of IRC) inside the software where people can chat. Or it is rather the way i have to first discover the contact on clearnet and chat only with this contact or inside group established by this contact.

**Comment by Bilb** [comment, marked as answer]
You can do both. 
If you know someone who has Session they can send a message to your session ID and you will see it.
You can also join an opengroup to find people talking about the same subject.
Some of them are joinable from the app itself (join an opengroup button) and some of them have to be found by other means. Some people started to create directories. One of them is there https://sessiongroups.com/
